go.mod

```diff
@@ -1 +1 @@
-module github.com/Rhymond/go-money
+module github.com/danielwitz/go-money
```

Comment:
I don't think this one should be here

Reply:
Yeah, it so I can use my fork locally. 
You should omit it if you merge it.